src/main/java/ua/com/danit/exception/EntityExceptionHandler.java

```
@@ -0,0 +1,23 @@
+package ua.com.danit.exception;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class EntityExceptionHandler
+    extends ResponseEntityExceptionHandler {
+
+  @ExceptionHandler(value
+      = { IllegalArgumentException.class, IllegalStateException.class })
```

remove concrete exceptions classes, let it catch all exceptions